ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2508,6 +2509,33 @@ monitor_sessions:
       size: < bytes >
 ```
 
+#### Tap Aggregation
+
+```yaml
+tap_aggregation:
+  mode:
+    exclusive:
+      enabled: < true | false >
+      profile: < profile_name | default --> tap-aggregation-default >
```

Comment:
We don't support default values in eos_cli_config_gen. As this default value can change anytime in future.